main.ts

```diff
@@ -62,6 +62,16 @@ class Translator {
     this.emit(')');
   }
 
+  visitExternalModuleReferenceExpr(expr: ts.Expression) {
+    // TODO: what if this isn't a string literal?
+    var moduleName = <ts.StringLiteralExpression>expr;
+    if (moduleName.text.lastIndexOf("./", 0) !== 0) {
```

Comment:
Is this funky code for saying "does not start with"? What about "string.startsWith" from ES6, or just !string.match(/^.\//) ?